ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -70,8 +70,8 @@
  */
 class GetBlobOperation extends GetOperation {
   // the callback to use to complete the operation.
-  private final OperationCompleteCallback operationCompleteCallback;
-  // whether the operationCompleteCallback has been called already.
+  private final OperationCallback operationCallback;
+  // whether the operationCallback has been called already.
   private final AtomicBoolean operationCallbackInvoked = new AtomicBoolean(false);
```

Comment:
Good point. I have renamed `OperationCallback` as that term implies that the callback is the one associated with the operation. So this variable will stay as is, but the callback used to notify the router will be called `RouterCallback`.